### ISSUE TITLE
Filter issue: Several species are missing from the dropdown

### ISSUE THREAD

**Issue by LunaAsefaw** [kartoza/sawps#1431]
There are 14 species in the dropdown:
![image](https://github.com/kartoza/sawps/assets/90245715/2eab0325-6580-4726-9953-28f7e80b8184)

But there are 188 species in the backend.

**Comment by LunaAsefaw** [comment]
@zamuzakki This might not be a bug actually. I think the species filter is created such that it only displays species that are found on the properties that you have selected. That is why when you click on clear all, no properties are selected thus no species can be shown on the dropdown #1390 so currently only 14 species have data on the system. @timlinux are we happy with this workflow?

**Comment by zamuzakki** [comment]
Yes, the species and property filter are organisation dependant.

**Comment by zamuzakki** [comment]
Yes @LunaAsefaw , the species and property filter are organisation dependant.
